ckanext/gdi_userportal/logic/action/post.py

```diff
@@ -12,24 +13,36 @@
 )
 from typing import Dict
 
+log = logging.getLogger(__name__)
 
-@toolkit.side_effect_free
-def enhanced_package_search(context, data_dict=None) -> Dict:
-    if data_dict is None:
-        data_dict = toolkit.request.json
-
-    result = toolkit.get_action("package_search")(context, data_dict)
 
-    values_to_translate = collect_values_to_translate(result)
-    translations = get_translations(values_to_translate)
+def handle_exception(e: Exception, message: str):
```

Comment:
Nice improvement. What I miss In python is public, private , static, internal decorators.. In C#/Java mostly of functions the order is 
- public 
-private 
- static

Reply:
I agree that it's a pity that Python does not provide these access modifiers. 

I put an underscore before the function name. That's an accepted convention in Python to say it should be private